challenge_1/python/igniteflow/src/challenge_1.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
```

Comment:
I appreciate this bang. This is compatible with python 2.x and 3.x. Way to go! Although not with windows, but who cares? ;)

Just kidding. Good job! thank you for your submission.